examples/tensorflow/text-generator/README.md

```diff
@@ -74,7 +74,7 @@ $ cortex get generator
 
 We can use `curl` to test our prediction service:
 
-url: http://***.amazonaws.com/text/generator
```

Comment:
The output message for`$ cortex deploy` should be updated from `deployment started` to what it is now. Should be done for all examples.